### DISCUSSION TITLE
Markdown editor underline text from UI

### DISCUSSION THREAD

**Discussion by mrpavlitzky** [OrchardCMS/OrchardCore#12060] [Q&A]
There's no button for underlining text in the Markdown editor. I think it might make life easier for content creators who are not willing to use HTML in their text.

**Comment by hishamco** [comment]
There is no syntax for underline in Markdown, but you can write your custom shortcode that render it with the help of HTML

**Reply by mrpavlitzky** [reply]
I understand. There's a simple possiblity of using < u > < /u > html tags, but what I was thinking about was strictly an experience of an author who oftentimes does not want to get into tagging text and expects the most common formatting options to be available via UI.

**Reply by hishamco** [reply]
Many think there are many formatting options out there, but markdown is different than HTML :) BTW there are a lot of configuration that can be applied to the editor. Please check our [docs](https://docs.orchardcore.net/en/latest/docs/reference/modules/Markdown/#markdown-configuration)

**Reply by Skrypt** [reply]
Underline is not allowed in Markdown because it is reserved for actual "links" to remove any confusion. Per design.

**Reply by ns8482e** [reply]
You have to use [link](#) `[link](#) `  :) cheers
